### ISSUE TITLE
Update Accessibility on New Components 

### ISSUE THREAD

**Issue by subaha-cob** [CityOfBoston/patterns#588]
Accessibility should be access again to make sure new components pass accessibility tests.

**Comment by subaha-cob** [comment]
Including aria-labels, tags, etc
